api_app/models/domain/airlock_request.py

```diff
@@ -0,0 +1,42 @@
+from typing import List
+from enum import Enum
+from pydantic import Field
+from resources import strings
+from models.domain.airlock_resource import AirlockResource, AirlockResourceType
+
+
+class AirlockRequestStatus(str, Enum):
+    """
+    Airlock Request status
+    """
+    Draft = strings.AIRLOCK_RESOURCE_STATUS_DRAFT
+    Submitted = strings.AIRLOCK_RESOURCE_STATUS_SUBMITTED
+    InReview = strings.AIRLOCK_RESOURCE_STATUS_INREVIEW
+    Approved = strings.AIRLOCK_RESOURCE_STATUS_APPROVED
+    Rejected = strings.AIRLOCK_RESOURCE_STATUS_REJECTED
+    Cancelled = strings.AIRLOCK_RESOURCE_STATUS_CANCELLED
+    Blocked = strings.AIRLOCK_RESOURCE_STATUS_BLOCKED
+
+
+class AirlockContainer(str, Enum):
+    containerId: str = Field("", title="Container ID")
+    type: AirlockRequestStatus = Field("", title="Container type")
+    connectionString: str = Field("", title="Container connection string")
+
+
+class AirlockRequestType(str, Enum):
+    Import = strings.AIRLOCK_REQUEST_TYPE_IMPORT
+    Export = strings.AIRLOCK_REQUEST_TYPE_EXPORT
+
+
+class AirlockRequest(AirlockResource):
+    """
+    Airlock request
+    """
+    workspaceId: str = Field("", title="Workspace ID", description="Service target Workspace id")
+    resourceType = AirlockResourceType.AirlockRequest
+    requestType: AirlockRequestType = Field("", title="Airlock request type")
+    files: List[str] = Field([], title="Files of the request")
+    business_justification: str = Field("Business Justifications", title="Explanation that will be provided to the request reviewer")
+    status: AirlockRequestStatus = Field(AirlockRequestStatus.Draft, title="Airlock request status")
+    container: List[AirlockContainer] = Field([], title="Containers of the request")
```

Comment:
I think the containers list should not be part of the Request schema.
can we add this just for the RequestInResponse?wdyt

Reply:
Was removed